persistence/javadsl/src/main/java/com/lightbend/lagom/javadsl/persistence/AkkaTaggerAdapter.java

```diff
@@ -21,13 +22,23 @@
   public static <Command, Event extends AggregateEvent<Event>>
       Function<Event, Set<String>> fromLagom(
           EntityContext<Command> entityContext, AggregateEventTagger<Event> lagomTagger) {
+    return fromLagom(entityContext.getEntityId(), lagomTagger);
+  }
+
```

Comment:
My idea is to fully remove `fromLagom(EntityContext, AggregateEventTagger)` but I could be convinced otherwise.